command/flag/deployment_strategy.go

```diff
@@ -23,13 +23,14 @@ func (h *DeploymentStrategy) UnmarshalFlag(val string) error {
 	case string(constant.DeploymentStrategyDefault):
 		// Do nothing, leave the default value
 
-	case string(constant.DeploymentStrategyRolling):
+	case string(constant.DeploymentStrategyRolling),
+		string(constant.DeploymentStrategyCanary):
 		h.Name = constant.DeploymentStrategy(valLower)
 
 	default:
 		return &flags.Error{
 			Type:    flags.ErrInvalidChoice,
-			Message: `STRATEGY must be "rolling" or not set`,
+			Message: `STRATEGY must be "canary", "rolling" or not set`,
```

Comment:
**non-blocking/thought** I'd favour letting the API decide what is a valid strategy rather than enumerating it in the CLI.